src/main/java/dk/bec/polonez/reservationsystem/dto/offerDto/ResponseFeatureDto.java

```diff
@@ -7,6 +7,8 @@
 @Builder
 public class ResponseFeatureDto {
```

Comment:
**advice** Package name is wrong (upper-case is not allowed in package name according to Java convention). BTW, is it correct that ```ResponseFeatureDto``` is in offer-related package (```xxx.yyy.offerDto```) instead of some ```zzz.feature``` package?